package.json

```diff
@@ -9,6 +9,12 @@
     "test":
       "lerna run --stream --scope zapier-platform-cli --scope zapier-platform-core --scope zapier-platform-schema --scope zapier-platform-legacy-scripting-runner test",
     "smoke-test":
-      "lerna run --stream --scope zapier-platform-cli --scope zapier-platform-core --scope zapier-platform-schema --scope zapier-platform-legacy-scripting-runner smoke-test"
+      "lerna run --stream --scope zapier-platform-cli --scope zapier-platform-core --scope zapier-platform-schema --scope zapier-platform-legacy-scripting-runner smoke-test",
+    "bump":
+      "lerna version --exact --force-publish=zapier-platform-cli,zapier-platform-core,zapier-platform-schema",
```

Comment:
Since i'm scared to actually run and test this command, can you talk a little about what actually happens when this is run?

My main concern is if we'll be able to easily version packages (mostly) individually. I think these will be our most common release patterns:

1. `core` alone: `9.0.1` -> `9.0.2` with no change in the schema dep
1. maj/min/patch release of `schema`, which causes the same in `core` (while depending on the new version). Eg, `schema` goes `9.0.1` -> `9.0.2`, `core` goes from `9.0.2` -> `9.0.3` w/ a bugfix and updated schema dependency
3. major version of cli (`8` -> `9`), no change in cli / core

Are are all of those supported and straightfoward with this command?

Reply:
Separate versioning is also something I love to see happening, but I narrowed down the scope of this PR, so it only deals with the current situation, which is to version altogether. On the operating level, I think it should be easy to do separate versioning. We probably only need to remove that `--force-publish` part from the `yarn bump` command. The hard part is the application-level dependencies, how we find and remove them in each package.